src/ModulesList.jsx

```diff
@@ -4,20 +4,11 @@ import Module from './Module.jsx';
 
 import fetch_modules from './fetch_modules';
 
-
-function getToken () {
-  try {
-    return GETVal('token');
-  } catch (e) {
-    return null;
-  }
-}
-
-function getUrl () {
+function getNodeVersion() {
   try {
-    return GETVal('url');
+    return GETVal('node_version') == '4' ? 4 : 8;
   } catch (e) {
-    return null;
+    return '8';
```

Comment:
Seems like we're returning two different types? An integer on line 9, and a string here.

`fetch_modules` checks for the integer. let's stay consistent.
```
const url = nodeVersion == 4 ? ...
```